app/components/activity/template.html

```diff
@@ -0,0 +1,83 @@
+<ul class="ph-timeline" ng-if="$ctrl.activities.length">
+  <li ng-repeat="activity in $ctrl.activities" class="ph-timeline-inverted">
+    <div class="ph-timeline-badge"
+      ng-if="activity.activityType === 'updateReply' || activity.activityType === 'createReply'"
+    >
+      <i class="fa fa-reply" aria-hidden="true"></i>
+    </div>
+    <div class="ph-timeline-badge"
+      ng-if="activity.activityType === 'updateDiscussion' || activity.activityType === 'createDiscussion'"
+    >
+      <i class="fa fa-comment" aria-hidden="true"></i>
+    </div>
+    <div ng-if="activity.activityType === 'documentHived'" class="ph-timeline-badge">
+      <img alt="logo white" height="30" src="./static/img/logo-hexagon-white.svg" />
+    </div>
+    <div class="ph-timeline-panel">
+      <div class="ph-sm-margin-bottom">
+        <small class="text-muted">
+          <i class="fa fa-clock-o" aria-hidden="true"></i>
+          <span am-time-ago="activity.updatedAt"></span>
+        </small>
+      </div>
+      <div>
+        <span ng-if="activity.person.account.avatar.type === 'google'">
```

Comment:
The avatar should be rendered with the `<avatar>` component, see https://github.com/paperhive/paperhive-frontend/blob/master/app/components/margin-discussion/template.html#L5